lib/membrane/video_compositor/compile/download_compositor.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Mix.Tasks.Compile.DownloadCompositor do
+  @moduledoc false
+  # Downloads VideoCompositor binaries.
+
+  use Mix.Task
+  require Membrane.Logger
+
+  @vc_version "v0.1.0-rc.2"
+  @membrane_video_compositor_plugin_path File.cwd!()
+
+  @impl Mix.Task
+  def run(_args) do
+    url =
+      "https://github.com/membraneframework/video_compositor/releases/download/#{@vc_version}/video_compositor_#{system_architecture()}.tar.gz"
+
+    unless File.exists?(vc_app_directory()) do
+      File.mkdir_p!(vc_app_directory())
```

Comment:
```suggestion
    unless File.exists?(Path.join(vc_app_directory(), ".lock")) do
      File.rm_rf!(vc_app_directory())
      File.mkdir_p!(vc_app_directory())
```
and create that `.lock` file at the end